source/_integrations/unifi.markdown

```diff
@@ -112,6 +112,13 @@ Entities appear for each WLAN. Changing the state of WLAN will trigger a reconfi
 
 ## Sensor
 
+These Sensor are disabled by default, to enable them, click the "Configure" button on the Unifi-integration page and go to page 3/3. Here the different sensors can be enabled.
```

Comment:
```suggestion
These sensors are disabled by default. Enabling a sensor adds a new device to Home Assistant for every connected client in your network. This can add complexity to large networks.

1. To enable the sensors, on the Unifi integration page, select **Configure**. 
2. Go to page 3/3 and enable the sensors.
```